src/MessageHandler/Organization/SynchronizePackageHandler.php

```diff
@@ -30,5 +34,9 @@ public function __invoke(SynchronizePackage $message): void
         }
 
         $this->synchronizer->synchronize($package);
+
+        if ($package->isSynchronizedSuccessfully()) {
+            $this->messageBus->dispatch(new ScanPackage($message->id()));
+        }
```

Comment:
we can leave it this way, but ultimately we could assume that we want to handle it later through the event, it's a pity that the unsuccessful publication of messages on the queue would affect the synchronization status (and this will happen now)

Alternatively, you could temporarily fix it this way: https://symfony.com/doc/current/messenger/message-recorder.html